src/paradigma/heart_rate/heart_rate_analysis.py

```diff
@@ -19,6 +23,8 @@ def extract_signal_quality_features(df: pd.DataFrame, config: SignalQualityFeatu
     
     # Compute statistics of the spectral domain signals
     df_windowed = extract_spectral_domain_features(config, df_windowed)
+
+    df_windowed.drop(columns = ['green'], inplace=True)  # Drop the values channel since it is no longer needed
```

Comment:
`inplace=True` wordt over het algemeen afgeraden:
- https://towardsdatascience.com/why-you-should-probably-never-use-pandas-inplace-true-9f9f211849e4
- https://sourcery.ai/blog/pandas-inplace/

Reply:
Is het een kwestie van de parameter weghalen? En ben het eens met je laatste comment dat dit miss uberhaupt overbodig is gezien we wss de return aan willen passen!

Reply:
Je moet de transformatie nog wel assignen aan een object, bijv: `df_windowed = df_windowed.drop(columns=['green'])`